packages.json

```diff
@@ -4889,5 +4889,17 @@
     "web": "https://github.com/miere43/nim-registry",
     "license": "MIT",
     "method": "git"
+  },
+  {
+    "name": "luna.nim",
```

Comment:
Sorry, one more thing. You should just call your package ``luna``. Rename your ``.nimble`` file to ``luna.nimble`` as well and make sure the ``name`` field inside is set to ``luna``.